### PR TITLE
Test5

### PR DESCRIPTION
**Pull request by kesselborn**

### CODE REVIEW

File: http-fs.go [hunk 1]
```diff
@@ -101,9 +102,9 @@ func serve(dir string, readOnly bool) func(w http.ResponseWriter, r *http.Reques
 }
 
 func main() {
-	addrParam := flag.String("addr", "0.0.0.0:5000", "where to listen for connection")
+	addr_param := flag.String("addr", "0.0.0.0:5000", "where to listen for connection")
 	dir := flag.String("dir", ".", "which directory to take as a root")
-	readOnly := flag.Bool("read-only", false, "start server read only")
+	read_only := flag.Bool("read-only", false, "start server read only")
```
[the remark this answers is not in the document]
Comment (kjedamzik) [reply http-fs.go]: /woof

**Comment by kesselprow** [comment]
@kjedamzik: [dog image](https://random.dog/fefd192a-3a64-4107-aa71-99e79d6d63e5.JPG)

<details>

In response to [this](https://github.com/kesselprow2000/prow-testing/pull/8#discussion_r239112741):

>/woof
>


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: http-fs.go [hunk 2]
```diff
@@ -101,9 +102,9 @@ func serve(dir string, readOnly bool) func(w http.ResponseWriter, r *http.Reques
 }
 
 func main() {
-	addrParam := flag.String("addr", "0.0.0.0:5000", "where to listen for connection")
+	addr_param := flag.String("addr", "0.0.0.0:5000", "where to listen for connection")
```
[the remark this answers is not in the document]
Comment (kjedamzik) [reply http-fs.go]: /woof

**Comment by kesselprow** [comment]
@kjedamzik: [dog image](https://random.dog/fa14cf36-4f3e-4e88-a512-a10f1d4f1be5.JPG)

<details>

In response to [this](https://github.com/kesselprow2000/prow-testing/pull/8#discussion_r239112781):

>/woof


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by kesselprow** [comment]
@kjedamzik: [dog image](https://random.dog/26c6d088-05de-4b85-b446-e961e2b6c516.jpg)

<details>

In response to [this](https://github.com/kesselprow2000/prow-testing/pull/8#issuecomment-444526390):

>/woof
>/lint
>/woof


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by kesselprow** [comment]
@kjedamzik: [cat image](https://thecatapi.com/?image_id=2ml)

<details>

In response to [this](https://github.com/kesselprow2000/prow-testing/pull/8#issuecomment-444526680):

>/meow


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**